api/strategies/openidStrategy.spec.js

```
@@ -1,175 +1,301 @@
-      });
-      await validateFn(tokenset, userinfo, (err, user, details) => {
-        expect(err).toBe(null);
-        expect(user).toBe(false);
-        expect(details.message).toBe(
-          'You must have the "' + process.env.OPENID_REQUIRED_ROLE + '" role to log in.',
-        );
-      });
+    const userinfo = { ...baseUserinfo };
+
+    // Act
+    const { user } = await validate(tokenset, userinfo);
```

ESLint

[error] 'user' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)